x-pack/legacy/plugins/integrations_manager/public/screens/detail/readme.tsx

```diff
@@ -8,10 +8,30 @@ import { EuiLoadingContent, EuiText } from '@elastic/eui';
 import ReactMarkdown from 'react-markdown';
 import { getFileByPath } from '../../data';
 import { markdownRenderers } from './markdown_renderers';
+import { useLinks } from '../../hooks';
 
-export function Readme({ readmePath }: { readmePath: string }) {
+export function Readme({
+  readmePath,
+  packageName,
+  version,
+}: {
+  readmePath: string;
+  packageName: string;
+  version: string;
+}) {
   const [markdown, setMarkdown] = useState<string | undefined>(undefined);
 
+  const handleImageUri = React.useCallback(
+    (uri: string) => {
+      const { toRelativeImage } = useLinks();
+      const isRelative =
+        uri.indexOf('http://') === 0 || uri.indexOf('https://') === 0 ? false : true;
```

Comment:
```suggestion
        /^http(s?):/.test(uri)
```

Reply:
I was trying to avoid using a regex here, unless you see a case the condition is missing?

Reply:
No, I don't a missing case. Why did you want to avoid a regex? Legability? Something else?

I understand that regexes aren't obvious in many cases, but I think this one is pretty straightforward. Less subjectively, it has less statements and concepts than the multiple `indexOf`s.

[`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) is another option which makes the intent clear without using a regex

```js
uri.startsWith('http://') || uri.startsWith('https://') ? false : true;
```

I'm pretty sure corejs will polyfill this for IE, but we would want to confirm.

Reply:
I try to avoid them because they can be much slower and difficult to read.